src/components/Sidebar/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from 'react';
+import {
+  ActivePostContext,
+  CommentFormContext,
+  ErrorsContext,
+  LoaderContext,
+} from '../../../utils/Store';
+import { Loader } from '../../Loader';
+import { ErrorTypes } from '../../../types/ErrorTypes';
+import { CommentItem } from './CommentItem';
+import { Comment } from '../../../types/Comment';
+
+interface CommentListProps {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<CommentListProps> = ({ comments }) => {
+  const { activePost } = useContext(ActivePostContext);
+  const { isLoading } = useContext(LoaderContext);
```

Comment:
Consider renaming 'isLoading' to follow the checklist item: 'don't use isLoad, it can be isLoading or isLoaded'. If 'isLoading' is already used correctly, no change is needed.